cmake/installing.cmake

```diff
@@ -1,4 +1,14 @@
 # F3D Installation
+
+# Are we building a python wheel?
+if (DEFINED SKBUILD)
```

Comment:
where is SKBUILD defined ? Does not seems a standard cmake var.

Reply:
See https://scikit-build.readthedocs.io/en/latest/usage.html#how-to-test-if-scikit-build-is-driving-the-compilation